packages/core/http/core-http-server/src/router/route.ts

```diff
@@ -160,7 +160,24 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
     idleSocket?: number;
   };
 
-  /** A short, human-friendly description of this endpoint */
+  /**
+   * Human-friendly summary of this route, used for (OAS) documentation
+   *
+   * @example
+   * ```ts
+   * router.get({
+   *  path: '/api/foo/{id}',
+   *  access: 'public',
+   *  summary: `Retrieve foo resources given an ID`,
```

Comment:
We're recommending that folks use more common verbs and "retrieve" is one our linters catch so let's not use it in this example
```suggestion
   *  summary: `Get foo resources for an ID`,
```